_makeLambdaHandler.js

```diff
@@ -1,23 +1,22 @@
 'use strict';
 
-var Q = require('q');
+var Q = require('q'),
+    log = console.log.bind(console); // eslint-disable-line no-console
```

Comment:
should we wait until you figure out a solution to the problem you discovered today with this same change in APIGW utils?

Reply:
Yep, that's the reason I pulled this back in https://github.com/silvermine/lambda-utils/pull/5#issuecomment-447945357. I'm writing up an issue now. Will update this PR once that has been resolved.

Reply:
Issue created: https://github.com/silvermine/apigateway-utils/issues/22